syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -141,6 +143,20 @@ def data(self):
         else:
             return self.native_data
 
+    @property
+    def players(self):
```

Comment:
Q: Shouldn't this be already added? (I think this PR needs to be rebased on master?)